java/core/src/main/java/co/worklytics/psoxy/gateway/impl/ApiDataRequestHandler.java

```diff
@@ -876,17 +876,16 @@ private boolean isUtf8CompatibleEncoding(String encoding) {
     @SneakyThrows
     ByteArrayContent reverseRequestBodyTokenization(@NonNull String contentType, String body) {
         // JSON case: use ObjectMapper to parse request body and map decode ON every string value
-        if (contentType.equals(ContentType.APPLICATION_JSON.toString())) {
```

Comment:
This is returning `application/json; charset=UTF-8`; needed to use `getMimeType()` to get the right value on the enum object